modules/task/backend.go

```diff
@@ -118,7 +93,28 @@ func (b *inMemBackend) Name() string {
 // Start implements the Module interface
 func (b *inMemBackend) Start(ready chan<- struct{}) <-chan error {
 	b.isRunning = true
-	return make(chan error)
+	errorCh := make(chan error, 2)
```

Comment:
What do you mean 'we control' the behavior? It is exposed to a user through NewInMemBackend.. If it is called twice, is it ok to create a new channel, because someone else can hold a reference to the previously started error channel?